src/software/world/ball_state_with_timestamp_test.cpp

```diff
@@ -0,0 +1,71 @@
+#include "software/world/ball_state_with_timestamp.h"
+
+#include <gtest/gtest.h>
+
+TEST(BallStateWithTimestampTest, get_position)
+{
+    BallStateWithTimestamp ball_state(Point(1, -2.3), Vector(0, 0.4),
+                                      Timestamp::fromSeconds(1.1));
+    EXPECT_EQ(ball_state.position(), Point(1, -2.3));
+}
```

Comment:
I don't think we need to test inherited methods here, given that that's already covered in the `BallState` tests.

Reply:
removed unneeded tests